app/lib/screens/study/tasks/task_screen.dart

```diff
@@ -37,16 +37,33 @@ class _TaskScreenState extends State<TaskScreen> {
   Widget _buildTask() {
     switch (taskInstance.task) {
       case CheckmarkTask checkmarkTask:
```

Comment:
I was not able to create a CheckmarkTask in the designer. Can you explain me how to do it?

Reply:
Any intervention is a `CheckmarkTask`. See: https://github.com/hpi-studyu/studyu/blob/a99ad4b29d8554520b2bf9d89ee8a8a4ace78dbb/designer_v2/lib/features/design/interventions/intervention_task_form_data.dart#L45-L52